test/TestProjects/TestProjectWithResource/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
```

Comment:
FYI - I'm moving the "test\TestProjects" folder to a new top-level directory - "TestAssets". This will be merged shortly.